packages/cli/src/lib/bundler/server.ts

```diff
@@ -115,17 +118,24 @@ export async function serveBundle(options: ServeOptions) {
     Number(url.port) ||
     (url.protocol === 'https:' ? 443 : 80);
 
-  const paths = resolveBundlingPaths(options);
-  const pkgPath = paths.targetPackageJson;
-  const pkg = await fs.readJson(pkgPath);
+  const detectedModulesEntryPoint = await createDetectedModulesEntryPoint({
+    config: fullConfig,
+    targetPath: paths.targetPath,
+    watch() {
+      server?.invalidate();
+    },
+  });
+
   const config = await createConfig(paths, {
+    ...options,
     checksEnabled: options.checksEnabled,
     isDev: true,
     baseUrl: url,
     frontendConfig,
     getFrontendAppConfigs: () => {
       return latestFrontendAppConfigs;
     },
+    additionalEntryPoints: detectedModulesEntryPoint,
```

Comment:
yeah just gotta figure out if entrypoint is one or two words in general

Reply:
It's two words, anything else needs a fixing x)